<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashBoard</class>
 <widget class="QDialog" name="DashBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="topicPath">
     <property name="text">
      <string>$SYS/broker/uptime</string>
     </property>
     <property name="placeholderText">
      <string>path/to/topic</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="widgetType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>temperature</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>detector</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>lever</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>blank</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wattmeter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>humidity meter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>relay</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>788</width>
        <height>499</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="widgets_frame"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="addWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:pressed{
	background-color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>Add widget</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="update">
     <property name="text">
      <string>update</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="description">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionasd">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
